training/src/anemoi/training/config/model/graphtransformer.yaml

```diff
@@ -58,7 +58,7 @@ processor:
   mlp_hidden_ratio: 4 # GraphTransformer or Transformer only
   num_heads: 16 # GraphTransformer or Transformer only
   cpu_offload: ${model.cpu_offload}
-  qk_norm: False
+  qk_norm: True
```

Comment:
Why change the behaviour?

Reply:
Need to check this with Simon and Helen. Looking at the settings, the `qv_norm` is kept to true for the processor in both the transformer (ens and single) and graph_transformer (ens). So that's why I thought this could be an error and needed correction. I will check with them before merging